sdk/src/vec.rs

```diff
@@ -32,6 +47,17 @@ impl<T: IntoTryFromVal> Ord for Vec<T> {
     }
 }
 
+#[cfg(any(feature = "std", test, feature = "testutils"))]
+impl<T> Debug for Vec<T>
+where
+    T: IntoTryFromVal + Debug + Clone,
+    T::Error: Debug,
+{
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "Vec({:?})", self.iter().collect::<std::vec::Vec<_>>())
```

Comment:
Todo: remove dependence on std, as if just superficial here and being used as a convenience.